src/connection.ts

```diff
@@ -1103,15 +1103,16 @@ export class Connection extends EventEmitter {
         this.log.debug(`sending test packet of ${sourceAmount} delivered ${prepareAmount} (exchange rate: ${exchangeRate})`)
         if (prepareAmount.precision(true) >= maxDigits) {
           return {
+            maxSuccessAmount: sourceAmount,
```

Comment:
should we adjust this to be higher even if it isn't a higher precison? i.e. 100 vs. 900 have the same precision but this would return whichever one is later in the list of packets

Reply:
Good point, let me adjust it to take that into account.